### DISCUSSION TITLE
Can pixart-LCM train 512 or 256 images?

### DISCUSSION THREAD

**Discussion by zmy-08** [PixArt-alpha/PixArt-alpha#124] [Q&A]
I'm honored to use such a good job！Can pixart-LCM train 512 or 256 images? Due to the equipment, I can't train 1024 HD images. Can you specify the code? I'm very much looking forward to your reply.

**Comment by lawrence-cj** [comment]
Of course. I would recommend you to follow the guidance here: https://github.com/PixArt-alpha/PixArt-alpha/blob/master/asset/docs/pixart_lcm.md
